### DISCUSSION TITLE
Proposal for community growth (English version)

### DISCUSSION THREAD

**Discussion by rvergara** [adempiere/adempiere#4118] [General]
**Resumen ejecutivo**

In order to revitalize the ADempiere community, during the project scrum meetings, a few ideas have been suggested:

- Redefine the "ADempiere incorporation" 
- Redefine project roles
- Reestablish the project directory
- Define rules for assigning funds to development projects 
- Creation and structuring of web domain, you tube account and email domain. 
- Scheduling weekly working meeting to generate ADempiere videos and manuals.
- Scheduling weekly meetings in order to support PR's completion.
- Reestablishment of ADempiere project citizens.
- Definition of agile and XP methodologies as tools for ADempiere developments. 
- Activate ADempiere messages and news using social media channels.

**Propuesta para ampliar la comunidad**

My proposal (Alberto Abudinen's) for ADempiere community growth is to generate a pattern to classify ADempiere project citizens as follows:

- people that support the community by paying a token fee of less than USD$5 in order to receive a monthly ADempiere news publication containing project advances, summary of regular project meetings and directory decisions 
- Citizens that pàt a USD$15 or so fee to receive all of the above, an additional categorization as ADempiere citizen. The person will be listed as such in project publications and he/she could attend the video and manuals meetings and could vote in those meetings
- Citizens that pay a higher fee or are classified as high meritocracy members that would receive all of the above two categories plus would be able to be named in different project roles or even be part of the ADempiere directory. Citizens of this category would be able to vote on project role and directory elections

**Other proposals**

This section contains proposals from other people that currently participate in scrum meetings
  
**Final notes**

**Comment by sdgathman** [comment]
I'm just looking for Adempiere chat.  I found the Matrix room created on gitter.im, which is mirrored on matrix.org and now my homeserver as well, but the last comment says they are moving to discord.  The mentioned discord room does not exist either.  There doesn't seem to be an IRC channel on Libera.  

Why not just continue to use the Matrix room?  You don't need gitter accounts to use it.  You don't need to sign up anywhere.  Anyone capable of running an Adempiere instance can run a Matrix homeserver.  Newbies can sign up with a public registrations homeserver (one list is at joinmatrix.org).
